.pre-commit-config.yaml.jinja

```diff
@@ -10,7 +10,7 @@
 {%- elif odoo_version < 17 %}
   {%- set proj_rev.pylint_odoo = "v8.0.20" %}
 {%- else %}
-  {%- set proj_rev.pylint_odoo = "v9.0.4" %}
+  {%- set proj_rev.pylint_odoo = "v9.1.3" %}
```

Comment:
You should put this version only for 18, not for 17. Please change also the ruff version to be inline with OCA.

Reply:
Okay, I have already defined the appropriate version of pylint for v17 and v18-
AFAIK OCA is still using `ruff v0.1.3`

Reply:
Thank you!